ironfish-cli/src/multisigBroker/server.ts

```diff
@@ -270,8 +305,18 @@ export class MultisigServer {
 
     let broadcasted = 0
 
-    for (const client of this.clients.values()) {
-      if (client.sessionId !== sessionId) {
+    const session = this.sessions.get(sessionId)
+    if (!session) {
+      this.logger.debug(`Session ${sessionId} does not exist, broadcast failed`)
+      return
+    }
+
+    for (const clientId of session.clientIds) {
+      const client = this.clients.get(clientId)
+      if (!client) {
+        this.logger.debug(
+          `Client ${clientId} does not exist, but session ${sessionId} thinks it does`,
+        )
```

Comment:
Should we remove the client from the session in this case? We shouldn't get to this state, but it could prevent the session from being cleaned up